lintro/tools/definitions/bandit.py

```
@@ -468,15 +468,19 @@ def check(self, paths: list[str], options: dict[str, object]) -> ToolResult:
                 issues=issues,
+                parse_failures_count=0,
             )
 
         except (json.JSONDecodeError, ValueError) as e:
+            # Bandit is a security scanner: unparseable output must never be
+            # reported as a clean pass (see #1044).
             logger.error(f"Failed to parse bandit output: {e}")
             return ToolResult(
                 name=self.definition.name,
                 success=False,
                 output=(output or f"Failed to parse bandit output: {str(e)}"),
```

Ruff (0.15.20)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)